bitcoin/src/Bitcoin/Transaction/Taproot.hs

```diff
@@ -249,25 +228,27 @@ encodeTaprootWitness = \case
 -- | Verify that the script path spend is valid, except for script execution.
 verifyScriptPathData ::
     -- | Output key
-    PubKey ->
+    PubKeyXY ->
```

Comment:
Now that we have better BIP341 support we could actually change this to `PubKeyXO`.  This is what appears in a taproot transaction output, and we should use it directly.